byron/ledger/executable-spec/src/Ledger/Update.hs

```diff
@@ -83,7 +104,7 @@ data PParams = PParams -- TODO: this should be a module of @cs-ledger@.
   -- ^ Maximum (abstract) transaction size in words
   , _maxPropSz :: !Natural
   -- ^ Maximum (abstract) update proposal size in words
-  , _bkSgnCntT :: Double
+  , _bkSgnCntT :: !BkSgnCntT
```

Comment:
Thanks for introducing these `newtype` wrappers! Although I guess you did this not for the sake of adding extra type safety, but because you needed them for the goblins :)

Reply:
Yes, goblins was the main motivation but I was pleased to increase type safety a little 🙂